app/renderer/components/task_card.js

```diff
@@ -6,6 +6,7 @@ const TaskStatus = require('../../common/task_status');
 const ProgressSpinner = require('./progress_spinner');
 
 const Material = require('../materialize');
+const SweetAlert = require('../sweetalert');
```

Comment:
Is SweetAlert a good name? The idea of wrapping the library is that we have a unique interface to alerts that we can change in the future if needed, I'm not sure it makes sense to put the same name than the actual library.

And yes, I'm aware I did the same thing for materialize, which may lead to confusion, but most code in materialize.js is garbage that I [cluttered together](https://en.wikipedia.org/wiki/Diogenes_syndrome) anyway, This alert can be quite useful as a nice wrapper.